### PR TITLE
Added readonly attribute

### LINKED ISSUE #307: Expose `readonly` attribute on the `input` element
In case we want to display a prefilled amount, which should not be editable, we would like to set the `input` as `readonly.` So could it be possible to expose a `readonly` attribute?

### PR DESCRIPTION
**Pull request by gianlucatomasino**
#### Description
# "Added readonly attribute to input" Closes #307

#### Changes
* Upgraded @ember/test-helpers to v.5 to have that https://github.com/emberjs/ember-test-helpers/pull/779
* Added attribute, test and upgraded docs

### Checklist 
- [X] Obvisously, add your option

- [X] Do not forget to write inline documentation for your code in addon/components/amount-input

- [X] Add a test, probably in tests/integration/components/amount-input.js

- [X] Add a playground example in tests/dummy/app/pods/docs/components/amount-input/all-options/template

### CODE REVIEW

File: tests/integration/components/amount-input/component-test.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { module, test } from 'qunit'
 import { setupRenderingTest } from 'ember-qunit'
-import { blur, fillIn, render, typeIn } from '@ember/test-helpers'
+import { blur, fillIn, render, typeIn, pauseTest } from '@ember/test-helpers'
```
Comment (mrloop) [inline tests/integration/components/amount-input/component-test.js:3]: Can remove unneeded import, btw you can access pauseTest directly on the test context without an import `await this.pauseTest()`

Comment (mrloop) [inline tests/integration/components/amount-input/component-test.js:3]: Can remove unneeded import, btw you can access pauseTest directly on the test context without an import `await this.pauseTest()`

File: tests/index.html [hunk 2]
```diff
@@ -1,33 +1,36 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta charset="utf-8" />
```
Comment (mrloop) [inline tests/index.html:4]: Don't think the trailing slashes are required, I'd not add them

Comment (mrloop) [inline tests/index.html:4]: Don't think the trailing slashes are required, I'd not add them

**Comment by mrloop** [comment]
Thanks for the PR. 

Couple of minor comments on the git history. Would be nice to have a separate commit for the QUnit update and the last commit correcting the typo can be squashed.

File: tests/index.html (hunk 2 above)
Comment (gianlucatomasino) [reply tests/index.html:4]: Sorry, my vscode added them automatically. I will remove them

File: tests/integration/components/amount-input/component-test.js (hunk 1 above)
Comment (gianlucatomasino) [reply tests/integration/components/amount-input/component-test.js:3]: Thank for the tips. I will remove the unneeded import.

**Comment by gianlucatomasino** [comment]
I did it. I created a clean history

File: tests/index.html (hunk 2 above)
Comment (mrloop) [reply tests/index.html]: thank you!

**Comment by gianlucatomasino** [comment]
Hi. I've created a new pull request #409. I close this one.

**Pull request closed without merging**